CHANGELOG.adoc

```diff
@@ -115,6 +115,7 @@ The implemented fix does not allow to evaluate an AQL expression, but only to re
 - https://github.com/eclipse-sirius/sirius-web/issues/3206[#3206] [diagram] Fix an issue where precondition on tool was not respected during palette creation
 - https://github.com/eclipse-sirius/sirius-web/issues/3257[#3257] [diagram] Fix an issue that caused an infinite loop when selecting elements with shift.
 - https://github.com/eclipse-sirius/sirius-web/issues/3239[#3239] [form] Add missing reference widget view builder
+- https://github.com/eclipse-sirius/sirius-web/issues/3269[#3269] [diagram] Disables node dragging during multi selection
```

Comment:
We can still drag nodes, maybe something like
Disables dropping nodes during multi selection

Reply:
You're right.
I changed it